addons/wiki/models.py

```diff
@@ -244,6 +244,8 @@ def clone_wiki(self, node_id):
         clone.node = node
         clone.user = self.user
         clone.save()
+        # Find and set Guid created during clone/save
+        clone._id = clone.guids.first()._id
```

Comment:
Rather than setting this in multiple places, you might be able to add this to the `ensure_guid` function, which runs after every save: https://github.com/CenterForOpenScience/osf.io/blob/c67b78a0d2b3b08c6cc228f5bbff6ed144dc843d/osf/models/base.py#L519